content/reference/rest-api/templates/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Templates'
+weight: 30
+
+menu:
+  main:
+    identifier: 'rest-api-templates'
+    parent: 'rest-api'
+---
+
+For a list of [methods](#methods) for this resource, see the end of this page.
+
+# Resource Representation
+
+```json
+{
+  "catalogId": "a5c7c2c6-03c1-4338-8325-24ee5fbf31be",
+  "content": {
+    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.4.0/resources/schema.json",
+    "appliesTo": [
+      "bpmn:ServiceTask"
+    ],
+    "description": "",
+    "id": "4f50eb6b-89c7-4684-8b61-228af15cf60c",
+    "name": "My First Template",
+    "properties": [],
+    "scopes": []
+  },
+  "version": {
```

Comment:
I thought about aligning the format of both endpoints but decided against it for the following reasons:

1. `GET /pluginApi/modeler/v1/templates` does not clearly separate between fields that represent a template's content (like `id` or `properties`) and additional informational fields like `metadata` or `version`. To achieve this cleaner separation I thus put all the fields that constitute a template's content below `content`, the rest is located at the root level
1. changing the name of this field from `version` to `metadata` would also require moving `catalogId` here (which would mix it with the version object) or creating a nested object `metadata` that contains everything that is currently below `version` and `catalogId`

@CatalinaMoisuc Given that, would you still go for aligning both representations?

Reply:
I would also prefer this to be aligned. If there is a good reason for chaning the way templates are returned, we can discuss to apply this change also to the Modeler api, but I don't see a reason why this should be different.
Ideally, we could switch the Modeler plugin to also use the public API and drop the Modeler API (but this would be a future step afterwards)

Reply:
﻿> ﻿I would also prefer this to be aligned.

@volkergersabeck Do you imagine the resource representations of the modeler and public api to be 100% aligned or just "more similar"? I'm asking because I'm wondering if aligning this would entail
* changing the proposed date format (ISO 8601) to the less readable one we're currently using for the modeler api (timestamp in milliseconds)?
* returning data that makes sense in the modeler api but not the public api (e.g. `metadata.catalogOrganizationId` or `metadata.tags`)?

Reply:
As @volkergersabeck  said: The benefit of keeping them as similar as possible is that we could more easily drop the modeler API as a next step after releasing the public API and make the modeler use the public API.

That doesn't necessarily mean we have to return the exact same data, but similar. (e.g. for now we won't need catalogOrganizaionId or tags, but we could add them later on when we switch from modeler to public API; e.g. we could still keep the "metadata" as a field name etc.).

Reply:
While I understand this reasoning I think that we should aim to get this endpoint as "right" as possible with our first shot - the public api should ideally be even more persistent than the modeler api so I would love to already include improvements in the first iteration and not do this at a later point when we maybe switch the Modeler Plugin to the public api.

I think the best course of action is to talk about this in more depth in a dedicated meeting together with the points discussed in https://github.com/camunda/camunda-docs-cawemo/pull/118#discussion_r688290603 so we can finalize this draft in a timely manner.

Reply:
During our follow-up meeting (see https://docs.google.com/document/d/1hHLxyPrqvqORjocnoYJDBl22-_wpeMBR-iyx6YYuxbw/edit?usp=sharing) we realized that I didn't know about a feature of the Modeler that enables users to store element templates as .json file in the `resources/element-templates folder` of the Modeler (see https://github.com/camunda/camunda-modeler/tree/master/docs/element-templates#configuring-templates).

As users of the public API should benefit from the same mechanism (i.e. they should be able to just copy the output of the public api to the above-mentioned folder) we agreed on aligning the output of the modeler api and the public api with some additional information in the `metadata` field returned by the latter.